packages/datadog-plugin-amqp10/src/index.js

```diff
@@ -78,22 +78,19 @@ function startReceiveSpan (tracer, config, link) {
   return span
 }
 
-function addTags (tracer, config, span, link) {
-  const address = link.session.connection.address
+function addTags (tracer, config, span, link = {}) {
+  const address = getHost(link)
 
   span.addTags({
     'service.name': config.service || `${tracer._service}-amqp`,
     'span.type': 'worker',
     'amqp.link.name': link.name,
     'amqp.link.handle': link.handle,
     'amqp.connection.host': address.host,
-    'amqp.connection.port': address.port
+    'amqp.connection.port': address.port,
```

Comment:
`null` tags still get set and sent right? do we want these to possibly show as `null` or omit them?

Reply:
`null` and `undefined` are skipped by the formatter.